addons/OPT/GPS/fn_clientInit.sqf

```diff
@@ -46,10 +46,46 @@
 			((uiNamespace getVariable "RscCustomInfoMiniMap") displayCtrl 101) call CFUNC(registerMapControl);
 		};
 
+		if (!(isNull ((findDisplay 133) displayCtrl 101))) then
+		{
+			((findDisplay 133) displayCtrl 101) call CFUNC(registerMapControl);
+		};	
+
+		if (!(isNull ((findDisplay 133) displayCtrl 13301))) then
+		{
+			((findDisplay 133) displayCtrl 13301) call CFUNC(registerMapControl);
+		};	
 
     }, 1] call CFUNC(addPerFrameHandler);
 
 	// MiniGPS Steuerung Init
 	[] call FUNC(miniGPS);
 
+	// Kontinuierlicher Neustart des GPS System, Löschung alle Marker und Neuerstellung
+	[{
+
+		// When BuildFlag != Version, the cache should be rebuilt automatically.
+		// The cache is what is actually drawn.
+		// The MapGraphicsGroup is what is currently registered to be drawn
+		// The MapGraphicsMapControls contain all controls where graphics should be drawn on
+		DUMP(clib_MapGraphics_MapGraphicsCacheBuildFlag);
+		DUMP(clib_MapGraphics_MapGraphicsCacheVersion);
+		DUMP(clib_MapGraphics_MapGraphicsCache);
+		DUMP(uinamespace getVariable "clib_mapgraphics_MapGraphicsMapControls");
+		DUMP(clib_MapGraphics_MapGraphicsGroup call clib_fnc_allVariables);
+
+		// alle Marker Löschen der Einheiten
+		allPlayers apply 
+		{
+			[_x] call FUNC(removeUnitFromGPS);
+		};
+
+		// Marker neu erstellen für alle
+		allPlayers apply 
+		{
+			[_x] call FUNC(addUnitToGPS);
+		};	
```

Comment:
Wenn das so funktioniert, kann man danach testen, ob man die beiden Schleifen nicht auch in einem abwickeln kann. Da das ja aber nur alle 60s ausgeführt wird, sollte der Performanceverlust irrelevant sein :)